### ISSUE TITLE
Update to use VALVE

### ISSUE THREAD

**Issue by jamesaoverton** [jamesaoverton/hipc-gates#22]
I would like to add examples, terminology browser, and validation form for cell names and markers using VALVE, following the model of the extended immune exposure site:

- https://github.com/jamesaoverton/immune-exposure-validation-2
- https://docs.google.com/spreadsheets/d/1A62wTHoSUMZENldjhCZt7loyb_OFA_PbfsqKPpDjWx4/edit#gid=708146477
- https://immune-exposure.ontodev.com/

I've done preliminary work on a Google Sheet here: 

https://docs.google.com/spreadsheets/d/109FaxCDuwj9fxPqk1_haIrwo3_Y6EPU8lbNGYzmbRsU/edit#gid=0

I also have some data and scripts that I will shared.

**Comment by jamesaoverton** [comment]
Looking around at the code here and locally, we need to take a step back.

1. add a simplified table of "special_markers" from [here](https://docs.google.com/spreadsheets/d/1jCieXeH_T83d0K3n_3W8QFiCrN4AASw5FlP0nezPhOI/edit#gid=0) to the new Google Sheet [here](https://docs.google.com/spreadsheets/d/109FaxCDuwj9fxPqk1_haIrwo3_Y6EPU8lbNGYzmbRsU/edit#gid=0)
2. generate a table of all valid markers (with columns: ID, Name) from three sources
    - special_markers
    - Protein Ontology labels and exact synonyms
3. generate a table of cell names (with columns: ID, Name) from Cell Ontology labels and synonyms
4. finish the VALVE configuration tables
5. test on three datasets (emailed): HIPC, IMPACC, LJI
6. update the interesting ontological code here to connect to VALVE
7. then start working on a new website

The tables from 2 and maybe 3 will be too large to fit in a Google Sheet, so we need tasks in the `Makefile` to build them locally or on DROID. The current `Makefile` and `src/` have code for this but I think we should write new code to use RDFTab.

**Comment by beckyjackson** [comment]
> add a simplified table of "special_markers" from here to the new Google Sheet here

Does "special_markers" correspond with "Special Gates" of the first Google Sheet? Or is it a combination of sheets?

**Comment by jamesaoverton** [comment]
Yes, "Special Gates", but we probably don't need all the columns.

**Comment by beckyjackson** [comment]
Do you think we only need ID & Label?

**Comment by jamesaoverton** [comment]
We need IDs and synonyms for now, so I edited to say "ID and Name" columns.

**Comment by beckyjackson** [comment]
For tracking:
- [x] add a simplified table of "special_markers"
- [x] generate a table of all valid markers (with columns: ID, Name) from three sources (special_markers, PR labels, PR synonyms)
- [x] generate a table of cell names (with columns: ID, Name) from Cell Ontology labels and synonyms
- [x] finish the VALVE configuration tables
- [x] test on three datasets (emailed): HIPC, IMPACC, LJI
- [ ] update the interesting ontological code here to connect to VALVE

**Comment by beckyjackson** [comment]
> update the interesting ontological code here to connect to VALVE

Was this supposed to be a link? Or just the code in this repo?

**Comment by jamesaoverton** [comment]
The code in this repo which compares gating definitions to Cell Ontology logical axioms, making up most of the content of the current web pages. This is what I'm thinking of:

https://github.com/jamesaoverton/cell-name-and-marker-validator/blob/master/src/server.py#L229

It would be better to talk about this on a call.
